.changeset/empty-doors-brush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/database': patch
```

Comment:
Need to include `@firebase/database-compat1` here

Reply:
Done.